templates/webapi/test/link_context.html.ep

```diff
@@ -0,0 +1,23 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+<div class="container">
+        <p>
+            Test-Module:
+            
+                <tt><%= $contextpath %></tt>
+            
+        </p>
+        <textarea class='code' id='script'><%= $context %></textarea>
+
+        <script type="text/javascript">
+         var editor = CodeMirror.fromTextArea(document.getElementById("script"), {
+             lineNumbers: true,
+             readOnly: true,
+             lineWrapping: true
+         });
+        </script>
```

Comment:
Don't duplicate all that code. It is for instance possible to include templates within templates to avoid this.

Reply:
fixed